tests/sync/test_smb.py

```diff
@@ -85,48 +67,61 @@ class TestConnect:
     def plugin(self):
         return smb.SmbPlugin()
 
-    def test_connect_with_default_options(self, plugin):
-        url = 'smb://myauthdomain;myusername@myserver.test/some/path'
-        keyring.set_password('kitovu', url, 'some_password')
-        plugin.connect(url, {})
+    @pytest.fixture
+    def info(self):
+        """Get connection info (like we'd get it from a config) for tests."""
+        keyring.set_password('kitovu-smb', 'myusername\nmyauthdomain\nexample.com', 'some_password')
+        return {
+            'domain': 'myauthdomain',
+            'username': 'myusername',
+            'hostname': 'example.com',
+            'share': 'myshare',
+        }
+
+    def test_connect_with_default_options(self, plugin, info):
+        plugin.configure(info)
+        plugin.connect()
+
         assert plugin._connection.init_args == {
             'username': 'myusername',
             'password': 'some_password',
             'domain': 'myauthdomain',
             'my_name': 'my-local-host',
-            'remote_name': 'myserver.test',
+            'remote_name': 'example.com',
```

Comment:
I switched this to `example.com` because that's reserved for purposes like this in [RFC 2606](https://tools.ietf.org/html/rfc2606) - then I did actually read that RFC, and `.test` is also reserved. Oh well, whatever, doesn't matter much :wink: